frontend/website/tests/confidence.spec.ts

```
@@ -0,0 +1,10 @@
+import { test, expect } from '@playwright/test';
+import { BUSINESS } from "../src/atoms/constants";
+
+test('has title', async ({ page }) => {
+  await page.goto(BUSINESS.WEBSITE);
+
+  // Expect a title "to contain" a substring.
+  const re = new RegExp("Building and fixing teams and processes")
```

Biome

[error] Use a regular expression literal instead of the RegExp constructor.

Regular expression literals avoid some escaping required in a string literal, and are easier to analyze statically.
Safe fix: Use a literal notation instead.


(lint/complexity/useRegexLiterals)